editor/plugins/asset_library_editor_plugin.cpp

```diff
@@ -63,6 +63,21 @@ void EditorAssetLibraryItem::configure(const String &p_title, int p_asset_id, co
 	price->set_text(p_cost);
 }
 
+void EditorAssetLibraryItem::clamp_width(int p_max_width) {
+	int text_pixel_width = title->get_theme_default_font().ptr()->get_string_size(title->get_text()).x * EDSCALE;
+	if (text_pixel_width > p_max_width) {
+		// Truncate title text to within the current column width.
+		String full_text = title->get_text();
+		title->set_tooltip_text(full_text);
+
+		int text_len = full_text.length();
```

Comment:
No need to extract this into a variable, it's clear what `full_text.length()` is and it's short enough to be used directly in the next line.

Reply:
Fixed. Thanks!